src/mixins/amalgamation-mixin.ts

```diff
@@ -209,14 +209,16 @@ export default class AmalgamationMixin extends Vue {
   async refetchAmalgamatingBusinessesInfo (): Promise<void> {
     const fetchTingInfo = async (item: any): Promise<AmalgamatingBusinessIF> => {
       const tingBusiness = await this.fetchAmalgamatingBusinessInfo(item)
+      // *** TODO: need to improve this
```

Comment:
@JazzarKarim 

Foreign and/or unaffiliated (when staff) are still not handled correctly.

Note that, when staff, unaffiliated businesses can be fetched correctly, but when not staff, they cannot. This will have to be sorted out.

Reply:
OK, got it.

Reply:
New ticket? Or does this fit in "do this last" ticket?

I'm not planning to change this further in my ticket (too many things already).

Reply:
Excellent. Yes, please!